deployer/src/main/java/eu/seaclouds/policy/SeaCloudsInitializerPolicy.java

```diff
@@ -0,0 +1,544 @@
+                            "                                    \"column\": \"value\"," +
+                            "                                    \"function\": \"mean\"," +
+                            "                                    \"query\": \"select mean(value) from \\\"" + metricName + "\\\" where $timeFilter group by time($interval) order asc\"," +
+                            "                                    \"refId\": \"A\"," +
+                            "                                    \"series\": \"" + metricName + "\"" +
+                            "                               }]," +
+                            "                  \"timeFrom\": null," +
+                            "                  \"timeShift\": null," +
+                            "                  \"title\": \"" + metricName + "\"," +
+                            "                  \"tooltip\": { \"shared\": true, \"value_type\": \"cumulative\"}," +
+                            "                  \"type\": \"graph\"," +
+                            "                  \"x-axis\": true," +
+                            "                  \"y-axis\": true," +
+                            "                  \"y_formats\": [\"short\",\"short\"]" +
+                            "              }]," +
+                            "      \"title\": \"" + entity.getDisplayName() + "\"" +
+                            "},";
+            rowOptions += currentRow;
+        }
+
+        // Remove last ,
+        rowOptions = rowOptions.substring(0, rowOptions.length() - 1);
+        grafanaInitializerPayload = globalGrafanaOptionsJSON + rowOptions + "] } }";
+
+    }
+
+    @Override
+    public void setEntity(EntityLocal entity) {
+        super.setEntity(entity);
+
+        // SeaCloudsInitializerPolicy should only be attached to Application
+        if (!entity.getApplication().equals(entity)) {
+            //TODO: Check if this exception stops the deployment process
+            throw new RuntimeException("SeaCloudsInitializerPolicy must be attached to an application");
+        }
+
+        parseMonitoringRules();
+        parseAgreements();
+        setupObserverInitializerPayload();
+        setupGrafanaInitializerPayload();
+        // Overriding the default START/STOP behaivour to include the aditional tasks required by
+        // SeaClouds while keeping the original behaviour. It allows to initialize and clean-up SeaClouds
+        // components when an application is added/removed.
+        ((EntityInternal) entity).getMutableEntityType().addSensor(SLA_ID);
+        ((EntityInternal) entity).getMutableEntityType().addSensor(T4C_IDS);
+        ((EntityInternal) entity).getMutableEntityType().addEffector(newStartEffector());
+        ((EntityInternal) entity).getMutableEntityType().addEffector(newStopEffector());
+
+    }
+
+    private Effector<Void> newStartEffector() {
```

Comment:
@andreaturli  This is true for the Lifecycle.Running event, sadly Brooklyn destroys the management context when the entity reaches Lifecycle.STOPPED so no ConfigKey available in this step to be able to invoke the http requests to clean up everything. 

I had two workarounds in my mind, one is to copy all the CK into field variables, another one is the one I took, overriding the START/STOP behavior.